doc/source/ray-overview/getting-started.md

```diff
@@ -139,7 +139,7 @@ Use the trained model for batch prediction with a ``BatchPredictor``.
 Ray has a rich ecosystem of libraries and frameworks built on top of it. 
 Simply click on the dropdowns below to see examples of our most popular libraries.
 
-`````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> Data: Creating and Transforming Datastreams
+`````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> Data: Creating and Transforming Data
```

Comment:
```suggestion
`````{dropdown} <img src="images/ray_svg_logo.svg" alt="ray" width="50px"> Data: Loading and Transforming Data
```